src/plugin/context/mod.rs

```diff
@@ -22,13 +24,30 @@ use crate::prelude::{CollisionGroups, RapierRigidBodyHandle};
 use rapier::control::CharacterAutostep;
 use rapier::geometry::DefaultBroadPhase;
 
+/// Marker component for to access the default [`RapierContext`].
+///
+/// This is used by [`systemparams::DefaultRapierContextAccess`] and other default accesses
+/// to help with getting a reference to the correct RapierContext.
+///
+/// If you're making a library, you might be interested in [`RapierContextEntityLink`]
+/// and leverage a [`Query<&RapierContext>`] to find the correct [`RapierContext`] of an entity.
+#[derive(Component, Reflect, Debug, Clone, Copy)]
+pub struct DefaultRapierContext;
+
+/// This is a component applied to any entity containing a rapier handle component.
+/// The inner Entity referred to has the component [`RapierContext`] responsible for handling
+/// its rapier data.
+#[derive(Component, Reflect, Debug, Clone, Copy, PartialEq, Eq)]
+pub struct RapierContextEntityLink(pub Entity);
```

Comment:
explore a 

```rs
RapierContextEntityLink<T = DefaultRapierContext>;
```

Reply:
What is the usecase for this?

Reply:
users having 2 different "main" worlds can use a similar API without reimplementing their own, it's rather straightforward but a low cost to support.

The downside being that we use a trait (so a documentation reader has to read it more carefully) is balanced with its default value, and impact is very local.

Reply:
This implementation should absolutely support a dynamic number of worlds, to maintain feature parity with rapier. If someone wanted statically checked world accesses, the end user could even make their own types to represent the link entity types.

Reply:
agreed :)